packages/fxa-payments-server/public/locales/en-US/main.ftl

```diff
@@ -381,6 +381,8 @@ plan-details-list-price = List Price
 
 coupon-discount = Discount
 coupon-discount-applied = Discount Reward Applied
+coupon-promo-code = Promo Code
+coupon-promo-code-applied = Promo Code Applied
```

Comment:
```suggestion
# Promo code for a coupon that provides a discount when applied
coupon-promo-code = Promo Code
# Promo code for a coupon that provides a discount when applied
coupon-promo-code-applied = Promo Code Applied
```
Suggest adding some individual comments as "Promo" could result in incorrect translations without context.